include/ignition/utils/utils.hh

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
this redirect header points to nothing and it should be removed

Reply:
https://github.com/ignitionrobotics/ign-utils/pull/55/commits/7348ea2684afe3e61f56309f7e0c08a088917a01